### ISSUE TITLE
Training on custom data??

### ISSUE THREAD

**Issue by dhruvsharma717** [s3nh/pytorch-text-recognition#12]
Hi, I'd like to know if this repo can be used to train CRAFT on custom data using weakly supervised learning ?
I have images with a lot of text on them. All of the text has been tagged but at a sentence level, i.e.,
I have the bounding box information and ground truth label information but at a sentence level.
Can weakly supervised learning be used in this case ?
